engine-tests/src/tests/uniswap.rs

```diff
@@ -49,19 +49,19 @@ fn test_uniswap_exact_output() {
 
     let (_result, profile) =
         context.add_equal_liquidity(LIQUIDITY_AMOUNT.into(), &token_a, &token_b);
-    utils::assert_gas_bound(profile.all_gas(), 32);
+    utils::assert_gas_bound(profile.all_gas(), 23);
     let wasm_fraction = 100 * profile.wasm_gas() / profile.all_gas();
     assert!(
-        (40..=50).contains(&wasm_fraction),
+        (50..=60).contains(&wasm_fraction),
```

Comment:
Revisit after resolving the comment on `MockedExternalWithTrie::get_trie_nodes_count`